jenkins/jobs/vic-product-scenario/selenium_grid.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+set -x
+
+start_node () {
+    docker run -d --net ${NETWORK} -e HUB_HOST=${HUB_NM} -v /dev/shm:/dev/shm --name "$1" "$2"
+
+    for _ in $(seq 1 10); do
+        if [[ "$(docker logs "$1")" = *"The node is registered to the hub and ready to use"* ]]; then
+            echo "$1 node is up and ready to use";
+            return 0;
+        fi
+        sleep 3;
+    done
+}
+
+remove_all () {
+    echo "Kill any old selenium infrastructure..."
+    docker rm -f ${HUB_NM} ${NODE1_NM} ${NODE2_NM} ${NODE3_NM} ${NODE4_NM}
+    docker network prune -f
+}
+
+if [ $# -ne 2 ] ; then
+    echo "Usage: $0 create/remove suffix"
+    exit 1
+fi
+
+ACTION=$1
+SUFFIX=$2
```

Comment:
If we are ever able to run multiple scenario tests at once for the same branch would we need to name seleniumgrid with the build number?
Are we better off running multiple executors?

Reply:
Now we can only run test on vic-executor1 because previous version vic OVAs are stored in /vic-cache of vic-executor1 for upgarde tests. We should enhance this to have multiple executors and only 1 job can be run in an executor at the same time.